Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -68,17 +71,20 @@ def specialization(self):
 
 
 class SerializedType(object):
-    def __init__(self, struct_or_class, cf_type, namespace, name, parent_class_name, members, condition, attributes, templates, other_metadata=None):
+    def __init__(self, struct_or_class, cf_type, namespace, name, parent_class_name, members, dictionary_members, condition, attributes, templates, other_metadata=None):
         self.struct_or_class = struct_or_class
         self.cf_type = cf_type
         self.to_cf_method = None
         self.from_cf_method = None
         self.forward_declaration = None
+        self.custom_secure_coding_class = None
+        self.custom_secure_coding_header = None
```

Comment:
This doesn't need to be a member of the SerializedType class.  Headers are not necessarily associated with a type.  I think we should do something more like ConditionalHeader.webkit_platform for this instead.